smg-automotive/components-pkg
# feat: change vp header links for private sellers only

References [JIRA](https://smg-au.atlassian.net/browse/IN-2188)

## Motivation and context

Before the VP release to private sellers, we created redirects to point users to the new path. As this change has been tested and monitored without any issues, we want to update the header links to point towards the new URL.

## Before

Header links for private sellers VP point to `/{language}/member/vehcilepool`.

## After

Header links for private sellers VP point to `/{language}/vehicle-management`.

## How to test

Build the apps locally with `components-pkg` pre-release version (`23.1.0-replace-private-vp-link-in-navigation-headers-6a8bce6d1eb5b3c845c804f151e42172d9a5d159.1`), and confirm that private sellers VP header links are working as expected (both AS24 and MS24).
